docs/self-hosted/windows.rst

```diff
@@ -0,0 +1,76 @@
+.. _windows-install:
+
+=================
+Windows
+=================
```

Comment:
reStructuredText requires the headline armor to be aligned with the length of the section title.

```suggestion
=======
Windows
=======
```